test/functionalTest/harnessFunctions.sh

```diff
@@ -1239,7 +1244,6 @@ export -f brokerStart
 export -f localBrokerStop
 export -f localBrokerStart
 export -f brokerStop
-export -f localBrokerStop
```

Comment:
It makes sense... it was a duplicated line.

However, given that we are modifying this, probalby it is a good idea to take the opportunity to reorder them in this way:

brokerStart
brokerStrop
localBrokerStart
localBrokerStop